hack/tools/go.mod

```diff
@@ -56,9 +56,9 @@ require (
 	gopkg.in/inf.v0 v0.9.1 // indirect
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
-	k8s.io/api v0.29.11 // indirect
-	k8s.io/apiextensions-apiserver v0.29.11 // indirect
-	k8s.io/apimachinery v0.30.7 // indirect
+	k8s.io/api v0.29.12 // indirect
+	k8s.io/apiextensions-apiserver v0.29.12 // indirect
```

Comment:
This looks wrong. Why are they not v0.30 like the rest?

Reply:
Indirects are not usually bumped by dependabot. Even verify-release.sh does not check for mismatches in indirects.